www/scripts/main.js

```diff
@@ -229,6 +230,11 @@ function setLastRoomName(roomName) {
 
 /* Helpers */
 
+function displayMessage(recipient) {
```

Comment:
Might be worth renaming this function. It sounds like it's a function that's going to display the message. Rather than being a check.

```shouldDisplayMessage``` maybe? Not sure.